zarr/attrs.py

```diff
@@ -1,74 +1,81 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, print_function, division
-
-
 import json
-import os
-from threading import Lock
-import fasteners
+from collections import MutableMapping
 
 
-class PersistentAttributes(object):
+class Attributes(MutableMapping):
```

Comment:
Not very important, but this class could probably be stripped down if we wanted to.